provider/parameter.go

```diff
@@ -340,6 +346,12 @@ func parameterDataSource() *schema.Resource {
 				Optional:    true,
 				Description: "The order determines the position of a template parameter in the UI/CLI presentation. The lowest order is shown first and parameters with equal order are sorted by name (ascending order).",
 			},
+			"ephemeral": {
+				Type:        schema.TypeBool,
+				Default:     false,
+				Optional:    true,
+				Description: "The presence of the ephemeral dictates whether the parameter value will be preserved between consecutive workspace builds.",
```

Comment:
```suggestion
				Description: "The value of an ephemeral parameter will not be preserved between consecutive workspace builds.",
```

Small suggestion since I felt the original description left open what true/false means.

Reply:
Fixed